test/mochaContext.ts

```diff
@@ -69,6 +72,22 @@ export interface Fixtures {
       commitEpochTx: ContractTransaction;
       vestingManagerFactory: VestingManagerFactory;
       rewardWallet: RewardWallet;
+      hydraVault: HydraVault;
+    }>;
+  };
+  distributeVaultFundsFixture: {
+    (): Promise<{
+      hydraChain: HydraChain;
+      systemHydraChain: HydraChain;
+      bls: BLS;
+      hydraDelegation: HydraDelegation;
+      hydraStaking: HydraStaking;
+      liquidToken: LiquidityToken;
+      aprCalculator: APRCalculator;
+      distributeVaultFundsTx: ContractTransaction;
+      vestingManagerFactory: VestingManagerFactory;
+      rewardWallet: RewardWallet;
+      hydraVault: HydraVault;
```

Comment:
hydra vault is the contract we use but we have to use two different instances of it - one is DAOIncentiveVault, and the other is FeeHandler vault. Please, apply this logic in the namings.